### DISCUSSION TITLE
Openwrt系统日志中alist已成功挂载网盘，但无法进入IP:5244的页面

### DISCUSSION THREAD

**Discussion by lt161** [AlistGo/alist#2943] [Q&A]
使用的openwrt alist插件如下：
1 app-meta-alist_X.X.X-X_all.ipk         <---   https://istore.linkease.com/repo/all/meta/
2 alist_X.X.X-X_x86_64.ipk                   <---   https://istore.linkease.com/repo/x86_64/nas/
3 luci-app-alist_X.X.X-1_all.ipk             <---  https://istore.linkease.com/repo/all/nas_luci/
4 luci-i18n-alist-zh-cn_X.X.X-1_all.ipk  <---  https://istore.linkease.com/repo/all/nas_luci/

原来的老软路由A使用一切正常。今日换了一台软路由B(使用的openwrt版本和A机器完全相同)，将旧机器上的openwrt配置导入到新机器后，先是出现了alist无法启动的问题（即勾选luci页面的启动，保存启用后，提示：alist未启动）。该问题通过在启动项中启用alist启动项后，解决了。

但是却出现如下无法打开5244页面的问题。目前，在Openwrt的系统日志中，可以看到alist已经成功挂载网盘了，但是在浏览器输入：路由器IP:5244后却无法打开页面。

Openwrt的系统日志如下：
Sat Jan  7 21:08:53 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:53] reading config file: /etc/alist/config.json
Sat Jan  7 21:08:53 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:53] load config from env with prefix: ALIST_
Sat Jan  7 21:08:53 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:53] init logrus...
Sat Jan  7 21:08:54 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:54] Aria2 not ready.
Sat Jan  7 21:08:54 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:54] start server @ 192.168.0.110:5244
Sat Jan  7 21:08:55 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:55] success load storage: [/阿里云盘], driver: [Aliyundrive]
Sat Jan  7 21:08:56 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:56] success load storage: [/迅雷网盘], driver: [Thunder]
Sat Jan  7 21:08:57 2023 daemon.err alist[31928]: [36mINFO[0m[2023-01-07 21:08:57] success load storage: [/天翼云盘2], driver: [189Cloud]

浏览器输入IP:5244的提示
![微信截图_20230107215820](https://user-images.githubusercontent.com/122161849/211154517-0f9ae4c8-c781-4afd-89cb-8e7ab6f4b323.png)

**Comment by jadetexy** [comment]
挂载alist访问是  群晖IP+5244，如果群晖的IP本身是带端口的例如3000，怎么办，IP:3000:5244？

**Reply by NovaShen0724** [reply]
你的群晖是直接获取ip的 还是只是从路由映射了一个端口进来

**Comment by NovaShen0724** [comment]
看看端口占用？

**Reply by lt161** [reply]
确认就是alist在占用
